### DISCUSSION TITLE
Confusion about Include and select API

### DISCUSSION THREAD

**Discussion by ClementineZou** [prisma/prisma#13694] [Q&A] [answered]
Hi everyone, I am a beginner and I read document but have confusion: What's the meaning of “include” and what is the difference between “select” and “include”. Thank you

**Comment by richardwardza** [comment, marked as answer]
I'm stealing this directly from [nikolasburk's reply in slack](https://prisma.slack.com/archives/CA491RJH0/p1654687757964359?thread_ts=1654687060.254269&cid=CA491RJH0):

```
Hey Aurora. :wave:

- include is only for relation fields and is used when you want to additionally include relations in addition to all the scalar fields that are returned by default

- select is used when you want to only return certain fields but exclude other ones (like a password field)

This page in the docs should give you more info about this as well :slightly_smiling_face: https://www.prisma.io/docs/concepts/components/prisma-client/select-fields#return-the-default-selection-set

```

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
